plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/dialects/SQLDialectDescriptor.java

```diff
@@ -0,0 +1,518 @@
+        if (parentDialect != null) {
+            nestedTypes.addAll(parentDialect.getDataTypes(true));
+        }
+        return getTransformedValues(WordType.ATTR_DIALECTS_VALUE_TYPES, nestedTypes);
+
+    }
+
+    @Override
+    @NotNull
+    public Set<String> getFunctions(boolean nested) {
+        if (!nested) {
+            return getTransformedValues(WordType.ATTR_DIALECTS_VALUE_FUNCTIONS, functions);
+        }
+        HashSet<String> nestedFunctions = new HashSet<>(functions);
+        if (parentDialect != null) {
+            nestedFunctions.addAll(parentDialect.getFunctions(true));
+        }
+        return getTransformedValues(WordType.ATTR_DIALECTS_VALUE_FUNCTIONS, nestedFunctions);
+
+    }
+
+    @Override
+    @NotNull
+    public Set<String> getDDLKeywords(boolean nested) {
+        if (!nested) {
+            return getTransformedValues(WordType.ATTR_DIALECTS_VALUE_KEYWORDS_DDL, ddlKeywords);
+        }
+        HashSet<String> nestedDDLKeywords = new HashSet<>(ddlKeywords);
+        if (parentDialect != null) {
+            nestedDDLKeywords.addAll(parentDialect.getDDLKeywords(true));
+        }
+        return getTransformedValues(WordType.ATTR_DIALECTS_VALUE_KEYWORDS_DDL, nestedDDLKeywords);
+    }
+
+    @NotNull
+    @Override
+    public Set<String> getDMLKeywords(boolean nested) {
+        if (!nested) {
+            return getTransformedValues(WordType.ATTR_DIALECTS_VALUE_KEYWORDS_DML, dmlKeywords);
+        }
+        HashSet<String> nestedDMLKeywords = new HashSet<>(dmlKeywords);
+        if (parentDialect != null) {
+            nestedDMLKeywords.addAll(parentDialect.getDMLKeywords(true));
+        }
+        return getTransformedValues(WordType.ATTR_DIALECTS_VALUE_KEYWORDS_DML, nestedDMLKeywords);
+
+    }
+
+    @NotNull
+    @Override
+    public Set<String> getExecuteKeywords(boolean nested) {
+        if (!nested) {
+            return getTransformedValues(WordType.ATTR_DIALECTS_VALUE_KEYWORDS_EXEC, execKeywords);
+        }
+        HashSet<String> nestedExecuteKeywords = new HashSet<>(execKeywords);
+        if (parentDialect != null) {
+            nestedExecuteKeywords.addAll(parentDialect.getExecuteKeywords(true));
+        }
+        return getTransformedValues(WordType.ATTR_DIALECTS_VALUE_KEYWORDS_EXEC, nestedExecuteKeywords);
+    }
+
+    @Override
+    @NotNull
+    public Set<String> getTransactionKeywords(boolean nested) {
+        if (!nested) {
+            return getTransformedValues(WordType.ATTR_DIALECTS_TXN_KEYWORDS, txnKeywords);
+        }
+        HashSet<String> nestedTXNKeywords = new HashSet<>(execKeywords);
+        if (parentDialect != null) {
+            nestedTXNKeywords.addAll(parentDialect.getTransactionKeywords(true));
+        }
+        return getTransformedValues(WordType.ATTR_DIALECTS_TXN_KEYWORDS, nestedTXNKeywords);
+    }
+
+    @Override
+    @NotNull
+    public String getScriptDelimiter() {
+        return CommonUtils.isEmpty(scriptDelimiter) ? ";" : scriptDelimiter;
```

Comment:
It doesn't seem to be used somewhere. Also I don't like the idea of duplicating the setting about statements delimiter.